runtime/cpp/yolov5seg/yolov5seg_example.cpp

```diff
@@ -0,0 +1,405 @@
+    }
+    std::cout << BOLDMAGENTA << "-I-----------------------------------------------\n" << std::endl << RESET;
+}
+
+
+Expected<std::shared_ptr<ConfiguredNetworkGroup>> configure_network_group(VDevice &vdevice, std::string yolov_hef)
+{
+    auto hef_exp = Hef::create(yolov_hef);
+    if (!hef_exp) {
+        return make_unexpected(hef_exp.status());
+    }
+    auto hef = hef_exp.release();
+
+    auto configure_params = hef.create_configure_params(HAILO_STREAM_INTERFACE_PCIE);
+    if (!configure_params) {
+        return make_unexpected(configure_params.status());
+    }
+
+    auto network_groups = vdevice.configure(hef, configure_params.value());
+    if (!network_groups) {
+        return make_unexpected(network_groups.status());
+    }
+
+    if (1 != network_groups->size()) {
+        std::cerr << "Invalid amount of network groups" << std::endl;
+        return make_unexpected(HAILO_INTERNAL_FAILURE);
+    }
+
+    return std::move(network_groups->at(0));
+}
+
+std::string getCmdOption(int argc, char *argv[], const std::string &option)
+{
+    std::string cmd;
+    for (int i = 1; i < argc; ++i)
+    {
+        std::string arg = argv[i];
+        if (0 == arg.find(option, 0))
+        {
+            std::size_t found = arg.find("=", 0) + 1;
+            cmd = arg.substr(found);
+            return cmd;
+        }
+    }
+    return cmd;
+}
+
+int main(int argc, char** argv) {
+
+    hailo_status status = HAILO_UNINITIALIZED;
+
+    std::chrono::duration<double> total_time;
+    std::chrono::time_point<std::chrono::system_clock> t_start = std::chrono::high_resolution_clock::now();
+
+    std::string yolo_hef       = getCmdOption(argc, argv, "-hef=");
+    std::string video_path      = getCmdOption(argc, argv, "-input=");
+
+    std::chrono::time_point<std::chrono::system_clock> write_time_vec;
+    std::chrono::duration<double> inference_time;
+    std::chrono::duration<double> postprocess_time;
+
+    auto vdevice_exp = VDevice::create();
+    if (!vdevice_exp) {
+        std::cerr << "Failed create vdevice, status = " << vdevice_exp.status() << std::endl;
+        return vdevice_exp.status();
+    }
+    auto vdevice = vdevice_exp.release();
+
+    auto network_group = configure_network_group(*vdevice, yolo_hef);
+    if (!network_group) {
+        std::cerr << "Failed to configure network group " << yolo_hef << std::endl;
+        return network_group.status();
+    }
+
+    auto input_vstreams_params = network_group.value()->make_input_vstream_params(QUANTIZED, FORMAT_TYPE_INPUT, HAILO_DEFAULT_VSTREAM_TIMEOUT_MS*100, HAILO_DEFAULT_VSTREAM_QUEUE_SIZE);
+    if (!input_vstreams_params) {
+        std::cerr << "Failed creating input vstreams " << input_vstreams_params.status() << std::endl;
+        return input_vstreams_params.status();
+    }
+
+    auto output_vstreams_params = network_group.value()->make_output_vstream_params(QUANTIZED, FORMAT_TYPE_OUTPUT, HAILO_DEFAULT_VSTREAM_TIMEOUT_MS*100, HAILO_DEFAULT_VSTREAM_QUEUE_SIZE);
```

Comment:
same params as input, use hailort api that creates both input & output together- it's shorter & clearer